### PR TITLE
fix(build): Provide libopen62541-tools package again

### PR DESCRIPTION
**Pull request by keba-estr**
Nodeset compiler has been removed with commit 9e0b766fbba73be8d703b6ccbfdeec5bfd200b48

### CODE REVIEW

**Comment by keba-estr** [comment]
Fixes issue #6406

**Comment by keba-estr** [comment]
@jpfr: Is there any version bump necessary before committing to a 1.4.1 hotfix branch?

**Review by jpfr — COMMENTED** [review]
Looks good. Comments inline.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1497,6 +1503,25 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
             DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 endif()
 
+set(UA_install_tools_dirs "tools/certs"
+    "tools/nodeset_compiler"
+    "tools/schema"
```
[the remark this answers is not in the document]
Comment (keba-estr) [reply CMakeLists.txt:1508]: I also use other files from the tools directory (and other sub directories than ua_nodeset):

Should the restriction only be for the folder usr/share/open62541/tools/ua-nodeset?

[the remark this answers is not in the document]
Comment (keba-estr) [reply CMakeLists.txt:1508]: @jpfr: The folder tools/schema already contains files named NodeIds.csv, StatusCode.csv and Opc.Ua.Types.bsd with a different content. It is not clear for me how to handle this in a compatible way. Could this be done in a separate PR?

**Pull request merged**